src/understand-tidb/the-lifecycle-of-a-statement.md

```diff
@@ -1 +1,67 @@
 # The Lifecycle of a Statement
+
+## MySQL protocol package with command and statement string
+
+After [connecting and getting authenticated](understand-tidb/mysql-protocol-and-session-management.md) the server is in a [statement execution loop](https://github.com/pingcap/tidb/blob/30cf15a59db11c34ffe05fc926152a43327eaa61/server/conn.go#L938) until the client is disconnected.
```

Comment:
7659bb3af39af1b56d8931e249d574d11d53be5d removed `understand-tidb/mysql-protocol-and-session-management.md`

Reply:
Updated. Thanks for the check!